web/packages/teleport/src/Discover/Shared/StatusLight.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Box } from 'design';
+import styled from 'styled-components';
+
+export enum ItemStatus {
+  Success,
+  Warning,
+  Error,
+}
+
+export const StatusLight = styled(Box)`
+  border-radius: 50%;
+  margin-right: 6px;
```

Comment:
```suggestion
  margin-right: ${props => props.theme.space[2]}px;
```

Let's use theme-provided spacing values to keep the UI looking consistent. This is going to use 8px.